ci-operator/step-registry/ipi/conf/nutanix/boot-categories-project/project/ipi-conf-nutanix-boot-categories-project-project-ref.yaml

```diff
@@ -0,0 +1,26 @@
+ref:
+  as: ipi-conf-nutanix-boot-categories-project-project
+  from_image:
+   namespace: ocp
+   name: "4.12"
+   tag: upi-installer
+  commands: ipi-conf-nutanix-boot-categories-project-project-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: PROJECT_NAME
+    default: "qe-project"
+    documentation: "project name"
+  - name: PROJECT_UUID
+    default: ""
+    documentation: "project uuid"
+  - name: PROJECT_TYPE
```

Comment:
We can set both `PROJECT_NAME` or `PROJECT_UUID`, or any of them?

Reply:
when PROJECT_TYPE set name, use PROJECT_NAME, when uuid, use PROJECT_UUID